truss/truss_handle.py

```diff
@@ -703,6 +703,8 @@ def update_python_version(self, python_version: str):
         self._update_config(
             lambda conf: replace(
                 conf,
+                # TODO: should this be wrapped in
+                #   `map_to_supported_python_version`?
```

Comment:
@squidarth Do you have any insight on whether this should be mapped here or explicitly have the ability to set it to *any* version?